### DISCUSSION TITLE
Setting up the prisma with plantscale driver adapter (Preview Feature)

### DISCUSSION THREAD

**Discussion by MarkForLoop** [prisma/prisma#22096] [Q&A]
### Question

I have been looking at how to use the preview feature for prisma (https://www.prisma.io/docs/guides/database/planetscale#how-to-use-the-planetscale-serverless-driver-with-prisma-preview)

when following the example it shows
`const prisma = new PrismaClient({ adapter })`
but when I follow this example my type-support shows that adapter is  

> Argument of type '{ adapter: PrismaPlanetScale; }' is not assignable to parameter of type 'Subset<PrismaClientOptions, PrismaClientOptions>'

because adapter doesnt exist in the PrismaClient is there a way to get the right type support without using a @ts-ignore

```
import { Client } from "@planetscale/database";
import { PrismaPlanetScale } from "@prisma/adapter-planetscale";
import { PrismaClient } from "@prisma/client";
import { env } from "~/env.mjs";
import { fetch as undiciFetch } from "undici";

const globalForPrisma = globalThis as unknown as {
  prisma: PrismaClient | undefined;
};

const client = new Client({
  url: env.DATABASE_URL,
  fetch: undiciFetch,
});
const adapter = new PrismaPlanetScale(client);
const prisma = globalForPrisma.prisma ?? new PrismaClient({});

// export const prisma = globalForPrisma.prisma ?? new PrismaClient();

if (env.NODE_ENV !== "production") globalForPrisma.prisma = prisma;
```

- OS: macOS
- Database:  PlanetScale
- Node.js version: 18.17.1
- Prisma                  : 5.6.0

**Comment by JClackett** [comment]
think their types are out of sync at the moment, think they have a beta release that fixes it

**Reply by JClackett** [reply]
and if you have issues with runtime type errors:
https://github.com/prisma/prisma/discussions/21347#discussioncomment-7576783

**Reply by JClackett** [reply]
but tbh, im just sticking to the adapter version: 5.5.0, until its fixed
